todo_app/app.py

```diff
@@ -16,9 +18,23 @@ def get_access_token_query_params():
 
 def create_app():
     app = Flask(__name__)
+
     app.config.from_object(Config())
+
+    app.config['LOG_LEVEL'] = os.getenv('LOG_LEVEL')
```

Comment:
Similar to above, we might want to provide a default value here that users can change, e.g.:
```suggestion
    app.config['LOG_LEVEL'] = os.getenv('LOG_LEVEL', 'DEBUG')
```